pkg/registry/apps/deployment/storage/storage.go

```diff
@@ -166,6 +166,10 @@ func (r *RollbackREST) Create(ctx context.Context, obj runtime.Object, createVal
 		return nil, errors.NewBadRequest(fmt.Sprintf("not a DeploymentRollback: %#v", obj))
 	}
 
+	if err := createValidation(obj.DeepCopyObject()); err != nil {
```

Comment:
the generic registry checks creationValidation for nil... would probably be good to do the same here

Reply:
added